src/Maestro/tests/Maestro.ScenarioTests/MaestroScenarioTestBase.cs

```diff
@@ -85,13 +86,40 @@ public string GetRepoFetchUrl(string org, string repository)
             return $"https://{_parameters.GitHubUser}:{_parameters.GitHubToken}@github.com/{org}/{repository}";
         }
 
+        public string GetAzDoRepoUrl(string repoName, string azdoAccount = "", string azdoProject = "")
```

Comment:
Would it make sense to instead make dncent/internal the defaults for azdoAccount and azdoProject?